npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/registry_finder.rb

```diff
@@ -121,8 +121,12 @@ def locked_registry
             known_registries.
             find { |h| h["registry"].include?(lockfile_registry) }&.
             fetch("registry")
+          return detailed_registry if detailed_registry
 
-          detailed_registry || lockfile_registry
+          uri = URI(registry_source_url)
+          return uri.host if uri.host == "npm.pkg.github.com" && uri.path&.start_with?("/download/")
```

Comment:
Could we assign this to a variable with a descriptive name before returning? I think that would help _me_ reason about this next time I read it 😄

Reply:
The new code removed the need for the variable at all 🙃